.github/workflows/publish.yaml

```diff
@@ -0,0 +1,103 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - 
+        name: Get tags
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+      -
+        name: Get Owner name
+        id: get_owner_name
+        run: echo "OWNER"=$(echo "$GITHUB_REPOSITORY" | awk -F / '{print $1}' | sed -e "s/:refs//") >> $GITHUB_ENV
+      -
+        name: Get Repository name
+        id: get_repo_name
+        run: echo ::set-output name=REPOSITORY_NAME::$(echo "$GITHUB_REPOSITORY" | awk -F / '{print $2}' | sed -e "s/:refs//")
+      - 
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: Login to Github Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
```

Comment:
why can't we use the default `${{ secrets.GITHUB_TOKEN }}` i feel like we discussed it in the wednesday call, but I can't remember why

Reply:
For the record: we tried it and later found out it doesn't work for forked repos.
https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token

Reply:
So what is `CR_PAT`?

Reply:
If it's not convention then can we use `DOCKER_PASSWORD` as per all the other merged and released builds?

Reply:
Its a secret for Container Registry.
I have seen CR_PAT in the Github Docs, thought it would be better if we go with it. What do you think?
https://docs.github.com/en/free-pro-team@latest/packages/getting-started-with-github-container-registry/migrating-to-github-container-registry-for-docker-images#authenticating-with-the-container-registry

Reply:
How faas-netes worked https://github.com/openfaas/faas-netes/tree/master/.github/workflows

is CR_PAT something that works without me doing something on each repo?

Reply:
That link doesn't reference `CR_PAT`, is it the right link?

Reply:
Sorry I can see where `CR_PAT` is mentioned now. It's not a special variable and has no meaning to Github Actions - can you change to `DOCKER_PASSWORD` please?

Reply:
Sure!